typed-racket-lib/typed-racket/typecheck/provide-handling.rkt

```diff
@@ -62,35 +66,38 @@
       [(dict-ref mapping internal-id
                  ;; if it wasn't there, put it in, and skip this case
                  (λ () (dict-set! mapping internal-id new-id) #f))
-       => (λ (mapped-id) (values #'(begin) mapped-id null))]
+       => (λ (mapped-id) (values #'(begin) #'(begin) mapped-id null))]
```

Comment:
This seem like it is the same as line 80. Maybe add a function: `(define (mk-ignored-quad id) (values #'(begin) #'begin) id null))`. Not sure if that would be cleaner though.